.github/workflows/ci-aarchxx-cross.yml

```diff
@@ -107,10 +107,14 @@ jobs:
         username: ${{secrets.DYNAMORIO_NOTIFICATION_EMAIL_USERNAME}}
         password: ${{secrets.DYNAMORIO_NOTIFICATION_EMAIL_PASSWORD}}
         subject: |
-          [${{github.repository}}] ${{github.workflow}} FAILED
+          [Author Action Required] [${{github.repository}}] ${{github.workflow}} FAILED
           on ${{github.event_name}} at ${{github.ref}}
         body: |
           Github Actions CI workflow run FAILED!
+          Commit author must triage and fix the possible regression.
+          Ensure the failure is not due to commits merged from out-of-date branches (i#5873).
+          For flaky tests, file new issue or update existing one with details.
```

Comment:
Add a request to re-run the failing job in order to try and keep the recorded and very visible post-merge commit status green.

Reply:
Done.